exercises/all-your-base/.meta/hints.md

```diff
@@ -0,0 +1,12 @@
+## Implementation
+
+Assumptions:
+1. Zero is always represented in outputs as [0] instead of [].
+2. In no other instances are leading zeroes present in any outputs.
+3. Leading zeroes are accepted in inputs.
+4. An empty sequence of input digits is considered zero, rather than an error.
+
+Handle problem cases with following messages:
+* input base must be >= 2
+* output base must be >= 2
+* all digits must satisfy 0 <= d < input base
```

Comment:
Since the test program does expect exact error match,
I'm wondering whether more specific Go wording in hints.md would be helpful.  Perhaps:
```
Handle problem cases by returning an error whose Error() method
returns one of these strings:
"input base must be >= 2"
"output base must be >= 2"
"all digits must satisfy 0 <= d < input base"
```
Reading the test program would lead to same conclusion. Your choice on this.

Reply:
I like the idea of hints as a more user-friendly way explaining expectations than using the test cases themselves. Although the test cases end up being the authority.

Reply:
@leenipper done so.